packages/audiodocs/docs/guides/see-your-sound.mdx

```diff
@@ -0,0 +1,262 @@
+---
+sidebar_position: 5
+---
+
+import InteractiveExample from '@site/src/components/InteractiveExample';
+
+# See your sound
+
+In this section, we explore the capabilities of the [`AnalyserNode`](/visualization/analyser-node) interface, focusing on techniques for extracting, processing, and visualizing audio data.
+Our exploration will lead us to create simple audio visualizer app.
```

Comment:
Can you make it a bit more blunt (less whee-whoo GPT excitement 😅). I don't have anything against using GPT as a base for writing the docs, but usually it generates a bit of noise-words making text feeling a bit less imformative

Reply:
Or to put this in other words: the simpler the language the better